MobileNav/SDL/SDL UI/Menu/SDLMenuButton.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SDLMenuButton.swift
+//  MobileNav
+//
+//  Created by James Lapinski on 4/14/20.
+//  Copyright © 2020 Livio Inc. All rights reserved.
+//
+
+import UIKit
+import SmartDeviceLink
+
+class SDLMenuButton: UIButton {
+
+    override func awakeFromNib() {
+       super.awakeFromNib()
+       setupButton()
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupButton()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupButton()
+    }
+
+    private func setupButton() {
+        backgroundColor = .systemPink
```

Comment:
Can we change this to a different color? I'm not fond of the pink. Maybe a dark blue?